internal/app/service_test.go

```diff
@@ -604,56 +605,46 @@ func TestLoadConfigurableFunctionPipelinesNumFunctions(t *testing.T) {
 	assert.Equal(t, expectedTransformsCount, len(pipeline.Transforms))
 }
 
-func TestUseTargetTypeOfByteArrayTrue(t *testing.T) {
+func TestTargetType(t *testing.T) {
 	functions := make(map[string]common.PipelineFunction)
 	functions["Compress"] = common.PipelineFunction{
 		Parameters: map[string]string{Algorithm: CompressGZIP},
 	}
 	functions["SetResponseData"] = common.PipelineFunction{}
-
-	sdk := Service{
-		lc: lc,
-		config: &common.ConfigurationStruct{
-			Writable: common.WritableInfo{
-				Pipeline: common.PipelineInfo{
-					ExecutionOrder:           "Compress, SetResponseData",
-					UseTargetTypeOfByteArray: true,
-					Functions:                functions,
-				},
-			},
-		},
-	}
-
-	_, err := sdk.LoadConfigurableFunctionPipelines()
-	require.NoError(t, err)
-	require.NotNil(t, sdk.targetType)
-	assert.Equal(t, reflect.Ptr, reflect.TypeOf(sdk.targetType).Kind())
-	assert.Equal(t, reflect.TypeOf([]byte{}).Kind(), reflect.TypeOf(sdk.targetType).Elem().Kind())
-}
-
-func TestUseTargetTypeOfByteArrayFalse(t *testing.T) {
-	functions := make(map[string]common.PipelineFunction)
-	functions["Compress"] = common.PipelineFunction{
-		Parameters: map[string]string{Algorithm: CompressGZIP},
+	tests := []struct {
+		name               string
+		targetTypeName     string
+		expectedTargetType interface{}
+	}{
+		{"raw", rawTargetType, &[]byte{}},
```

Comment:
Add a test for mixed case value.

Reply:
rawTargetType = raw

do you mean making the target time Raw or raW

Reply:
Yea or passing it `EveNt`

Reply:
I had to make some changes to the test, it broke when evaluating mixed case.